src/bindgen/config.rs

```diff
@@ -594,6 +594,8 @@ pub struct EnumConfig {
     /// Whether to generate empty, private default-constructors for tagged
     /// enums.
     pub private_default_tagged_enum_constructor: bool,
+    /// Whether to force all enums to use the C layout strategy, as if they have `[repr(C)]
+    pub force_repr_c: bool,
```

Comment:
So just to be clear, in our case this is actually not the behavior we want. We want to force `repr(C)` for `repr(u32)` and `repr(i32)`, otherwise this will create FFI boundary issues when a Rust `repr(u8)` enum gets treated as an `int` in C.

Reply:
yeah, good point. This works for us today because the only odd-one out is our internal bebop structure for input levels (`repr(u32)`), but agree this has potential to break in other cases. I guess we have size here which should have this info, let me see if there's a quick fix.

Reply:
swapped this to a vec, taking primitives. e.g. `i32` - think that addresses this concern.